api/object.go

```diff
@@ -216,14 +254,22 @@ func (opts UploadObjectOptions) Apply(values url.Values) {
 	if opts.ContractSet != "" {
 		values.Set("contractset", opts.ContractSet)
 	}
-	if opts.MimeType != "" {
-		values.Set("mimetype", opts.MimeType)
+	if ct, ok := opts.Metadata["Content-Type"]; ok {
```

Comment:
I think every mime type is a content type but not every content type essentially a valid mime type. So it might be dangerous to mix them up like this.